src/main_backend.py

```
@@ -168,9 +166,7 @@ def _initialize_tpu_devices(self) -> None:
         if not tpu_devices:
             return
 
-        from src.utils.device_management_utils.mx3_accelerator import (  # noqa: E402
-            MX3Accelerator,
-        )
+        from src.utils.device_management_utils.mx3_accelerator import MX3Accelerator  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)